src/server/handlers/login.js

```diff
@@ -118,8 +118,12 @@ export const logout = (req, res, next) => {
       // TODO log errors to sentry
       return next(new Error(constants.E_LOGOUT_FAIL));
     }
-    // FIXME redirect to page that initiated the sign in request
-    res.redirect('/');
+    if (process.env.NODE_ENV !== 'development') {
```

Comment:
Is this needed here? If home page is not redirecting further, redirecting to `/` (as it currently is) should be still fine(?)

Reply:
Thanks, I wasn't sure about this handler, as it seems only the github one is used